### ISSUE TITLE
Can not execute the 'Shfmt' command

### ISSUE THREAD

**Issue by 814182193** [z0mbix/vim-shfmt#10]
I installed the vim-shfmt plug through vim-plug,but when I want to format a bash script it doesn't work well,Please help me.My OS is Windows7,and Vim version is 8.2,By the 'PlugStatus' command,It's status is ok,but when i input the ':Shfmt' command,The Vim remind me that it's not a command of the editor,the error code is E492,Thank U!
![image](https://user-images.githubusercontent.com/17901065/123411567-2c275600-d5e3-11eb-9fc6-b9942244a81d.png)
![image](https://user-images.githubusercontent.com/17901065/123411818-7577a580-d5e3-11eb-84fd-132ead6a7ab2.png)

**Comment by farbodsz** [comment]
I had the same issue today.

Do you have the [shfmt](https://github.com/mvdan/sh) tool installed? (this plugin relies on it)

If so, are executable permissions set correctly?

**Comment by 814182193** [comment]
I'm sorry to reply you so long. I'm using it on windows. I've installed shfmt and configured it to environment variables,mvdan/sh for Windows is a single executable program. In the terminal, I need to enter the path and file name (including ". exe") to use it, for example: shfmt.exe - l - w my-test.sh,In this way, the file can be successfully formatted，I don't know how to associate vim-shfmt with mvdan/sh in windows and where to configure it;I also installed them in Ubuntu 20.04.2.0, which works well

**Comment by z0mbix** [comment]
Oh, this is a total oversight, as I’ve never tested it on Windows, so the code is hard coded to execute `shfmt`, not `shfmt.exe`. I will try to fix this soon, once I can sort out a windows VM to test with.

**Comment by 814182193** [comment]
Thank you very much and look forward to it!

**Comment by z0mbix** [comment]
I just realised that you can set this in your vimrc to set the binary name:

```
let g:shfmt_cmd = 'shfmt.exe'
```

**Comment by 814182193** [comment]
Thank you. I have solved this problem. It should be that shfmt is not installed correctly. I don't know how to use shfmt.exe because I don't know the Go language.It was developed in the Go language, Now I have installed the Go, configured the environment variables of the operating system, and put shfmt.exe in the bin directory under the Go installation path. In this way, I can use the "shfmt - l - w" command in my console,I don't need to enter the absolute path as before, including the suffix of the file which like "C:\Program Files\...\shfmt.exe -l -w", I also configured the parameters you said in the vimrc file。Anyway, now that it's working, I'm very happy and thank you all very much！
